flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/RestfulGateway.java

```diff
@@ -111,6 +112,14 @@
 	 */
 	CompletableFuture<ClusterOverview> requestClusterOverview(@RpcTimeout Time timeout);
 
+	/**
+	 * Requests the all the jobs overview.
```

Comment:
Because I saw the implementation of `requestClusterOverview ` combines the completed jobs' overview, but this method would not contains so I refer to `requestMultipleJobDetails`. All right, it seems there is no problem about the "all". Will change it.